app/src/main/java/org/mozilla/focus/fragment/FirstrunFragment.kt

```diff
@@ -134,7 +134,7 @@ class FirstrunFragment : Fragment(), View.OnClickListener {
 
     private fun promoteSetDefaultBrowserIfPreload(): FirstrunFragment {
         // if it's a system app(preload), we'll like to promote set default browser when the user finish first run
-        if (isSystemApp() && !AppConstants.isReleaseBuild()) {
```

Comment:
Maybe explain explicitly why we want non-release build to have this behavior too.